### PR TITLE
[DOC] Install apidoc, configure for one endpoint

### PR DESCRIPTION
**Pull request by stevenzeiler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ Node.js server application to track and publish rippled validator performance an
 
 a [Sails](http://sailsjs.org) application
 
+## API Documentation
+
+HTTP/JSON endpoint documentation is maintained in source control using [ApiDoc](http://apidocjs.com/)
+ApiDoc annotations exist above individual controller actions in `api/controllers/`
+
+````
+npm run apidoc
```
Comment (wilsonianb) [inline README.md]: I get `sh: 1: apidoc: not found`

Comment (stevenzeiler) [inline README.md]: `70fdf00` should fix the issue.

Comment (wilsonianb) [inline README.md]: `error: EACCES, permission denied 'apidoc/css/style.css'`

File: api/controllers/ValidationsController.js [hunk 2]
```diff
@@ -54,8 +54,19 @@ module.exports = {
...
+
+  /**
+   * @api {get} /validations 
+   * @apiName listValidations
+   * @apiGroup Validations
+   *
+   * @apiSuccess {Integer} validations_count total validations for validator in past 24 hours
+   * @apiSuccess {ValidationPublicKey} validation_public_key rippled validation public key
+  */
+   */
```
Comment (wilsonianb) [inline api/controllers/ValidationsController.js]: Remove extra `*/`

File: README.md (hunk 1 above)
Comment (stevenzeiler) [inline README.md]: Did the `EACCES` error turn out to be a local file system permissions error on your machine? I haven't experienced the issue.

Comment (wilsonianb) [inline README.md]: Yeah, the issue was on my end. Resolved

**Comment by wilsonianb** [comment]
LGTM, I think https://github.com/ripple/validator-registry-api/pull/47 should fix the circleci build

**Comment by stevenzeiler** [comment]
:+1:

**Pull request merged**